### DISCUSSION TITLE
Hacktoberfest topic

### DISCUSSION THREAD

**Discussion by rluvaton** [mobxjs/mobx#3131] [General]
It is that time again. hacktoberfest is totally opt in now:

> A repository/project is considered to be participating in Hacktoberfest if the **`hacktoberfest` topic is present** and is accepting public contributions via pull requests. An individual pull request can also be opted-in directly by adding the `hacktoberfest-accepted` label.

Could you please add the `hacktoberfest` topic?

**Comment by mweststrate** [comment]
Preferably not, from past experiences they attract primarily superficial
contributions, but often not fixes for actually open issues or significant
improvements in e.g docs. But if anything significant (50+ loc or a complex
infra update, and arguably benefiting the project) comes in, I'll happily
still award the label.

**Reply by rluvaton** [reply]
You can do what node doing https://github.com/nodejs/node/issues/40266
